editors/linux-wps-office/Makefile

```diff
@@ -0,0 +1,64 @@
+PORTNAME=	wps-office
+DISTVERSION=	11.1.0.10161
+CATEGORIES=	editors linux
+MASTER_SITES=	https://wdl1.pcfg.cache.wpscdn.com/wpsdl/wpsoffice/download/linux/10161/
+PKGNAMEPREFIX=	linux-
+DISTNAME=	${PORTNAME}-${PORTVERSION}.XA-1.x86_64
+
+MAINTAINER=	freebsd@sysctl.cz
+COMMENT=	Complete office suite with PDF editor
+
+LICENSE=	commercial
+LICENSE_NAME=	End User License Agreement
+LICENSE_TEXT=	${RESTRICTED}
+LICENSE_PERMS=	no-auto-accept no-dist-mirror no-dist-sell no-pkg-mirror no-pkg-sell
+
+RESTRICTED=	Source recipient must acknowledge the license. \
+		Reproduction or redistribution prohibited. \
+		See https://www.wps.com/eula.
+
+USES=		desktop-file-utils linux shebangfix
+INSTALLS_ICONS=	yes
+USE_LINUX=	png pulseaudio-libs xorglibs
+
+SHEBANG_FILES= 	usr/bin/et \
+		usr/bin/wpp \
+		usr/bin/wps \
+		usr/bin/wpspdf
+
+ONLY_FOR_ARCHS= amd64
+
+EXTRACT_SUFX=   .rpm
+NO_BUILD=	yes
+NO_WRKSUBDIR=	yes
+PLIST_SUB=	LINUXBASE=${LINUXBASE}
+
+WPS_BINS=	et wpp wps wpspdf
+WPS_DESK=	et wpp wps pdf prometheus
+
+post-patch:
+.for d in ${WPS_DESK}
+	@${REINPLACE_CMD} -e 's|/usr/bin|${PREFIX}/bin|' ${WRKSRC}/opt/kingsoft/${PORTNAME}/desktops/${PORTNAME}-${d}.desktop
+.endfor
+.for WPS_BINARY in ${WPS_BINS}
```

Comment:
Just being nitpicking, `WPS_BINARY` looks a global variable to me so I am confused at the first sight, especially it is longer than `WPS_BINS`. :-)

Just a preference/style question.

Reply:
For me I'll just use names like f, file, b, binary or so in lower case.

Reply:
Sure - this line is maintainer-submitted while I added the above one, will make them consistent.